crates/ruff_python_stdlib/src/builtins.rs

```diff
@@ -186,170 +23,343 @@ pub const MAGIC_GLOBALS: &[&str] = &[
+                | "SystemExit"
+                | "TabError"
+                | "TimeoutError"
+                | "True"
+                | "TypeError"
+                | "UnboundLocalError"
+                | "UnicodeDecodeError"
+                | "UnicodeEncodeError"
+                | "UnicodeError"
+                | "UnicodeTranslateError"
+                | "UnicodeWarning"
+                | "UserWarning"
+                | "ValueError"
+                | "Warning"
+                | "ZeroDivisionError"
+                | "__build_class__"
+                | "__debug__"
+                | "__doc__"
+                | "__import__"
+                | "__loader__"
+                | "__name__"
+                | "__package__"
+                | "__spec__"
+                | "abs"
+                | "all"
+                | "any"
+                | "ascii"
+                | "bin"
+                | "bool"
+                | "breakpoint"
+                | "bytearray"
+                | "bytes"
+                | "callable"
+                | "chr"
+                | "classmethod"
+                | "compile"
+                | "complex"
+                | "copyright"
+                | "credits"
+                | "delattr"
+                | "dict"
+                | "dir"
+                | "divmod"
+                | "enumerate"
+                | "eval"
+                | "exec"
+                | "exit"
+                | "filter"
+                | "float"
+                | "format"
+                | "frozenset"
+                | "getattr"
+                | "globals"
+                | "hasattr"
+                | "hash"
+                | "help"
+                | "hex"
+                | "id"
+                | "input"
+                | "int"
+                | "isinstance"
+                | "issubclass"
+                | "iter"
+                | "len"
+                | "license"
+                | "list"
+                | "locals"
+                | "map"
+                | "max"
+                | "memoryview"
+                | "min"
+                | "next"
+                | "object"
+                | "oct"
+                | "open"
+                | "ord"
+                | "pow"
+                | "print"
+                | "property"
+                | "quit"
+                | "range"
+                | "repr"
+                | "reversed"
+                | "round"
+                | "set"
+                | "setattr"
+                | "slice"
+                | "sorted"
+                | "staticmethod"
+                | "str"
+                | "sum"
+                | "super"
+                | "tuple"
+                | "type"
+                | "vars"
+                | "zip"
+        ) | (10..=13, "EncodingWarning" | "aiter" | "anext")
+            | (11..=13, "BaseExceptionGroup" | "ExceptionGroup")
+            | (13, "IncompleteInputError" | "PythonFinalizationError")
```

Comment:
What version of Python 3.13 do you have installed locally? I think `IncompleteInputError` was removed from builtins in June: https://github.com/python/cpython/pull/119680

Reply:
I have `3.13.0b1` -- thank you!